packages/account-v2/tailwind.config.ts

```diff
@@ -1,7 +1,23 @@
 import type { Config } from 'tailwindcss';
+const plugin = require('tailwindcss/plugin');
 
 export default {
     content: ['./src/**/*.{js,jsx,ts,tsx}'],
+    plugins: [
+        plugin(({ addUtilities }) => {
+            addUtilities({
+                '.backface-hidden': {
+                    'backface-visibility': 'hidden',
+                },
+                '.backface-visible': {
+                    'backface-visibility': 'visible',
+                },
+                '.d-none': {
```

Comment:
Currently `hidden` doesnt work from tailwind for some reason

Reply:
It's probably because of this class in our core css.
![image](https://github.com/binary-com/deriv-app/assets/135801848/cba2519d-9938-4de7-8335-8192799d2fb7)

Reply:
Oh yeah !. Nice catch